recipes/qt-msbuild/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "3.3.1":
+    url: "https://download.qt.io/official_releases/vsaddin/3.3.1/qt-vsaddin-msbuild-3.3.1.zip"
```

Comment:
Alternatively, we could get Qt/MSBuild from: https://github.com/qt-labs/vstools/tree/v3.3.1/QtMSBuild/QtMsBuild. I think they are identical to the ZIP distribution, although I haven't checked. But I think it's easier to just use the standalone download links.